web-app/js/portal/ui/FeatureInfoPopup.js

```diff
@@ -277,22 +277,20 @@ Portal.ui.FeatureInfoPopup = Ext.extend(GeoExt.Popup, {
     _updatePopupDepthStatus: function(response) {
         if (response && response.responseXML) {
             var xmldoc = response.responseXML;
-
-            // Depth service can return 204 but our app changes that to a 200 and pipes down nothing
-            if (xmldoc && xmldoc.getElementsByTagName('depth') !== undefined) {
-                var depth = xmldoc.getElementsByTagName('depth')[0].firstChild.nodeValue;
-                var label = OpenLayers.i18n(depth <= 0 ? 'depthLabel' : 'elevationLabel');
-
-                this.depthInfoPanel.update(this.locationString + " " + this._boldify(label) + " " + Math.abs(depth) + "m");
-            }
-            else {
-                this.depthInfoPanel.update("");
+            if (xmldoc) {
+                var depthTag = xmldoc.getElementsByTagName('depth')[0].firstChild;
```

Comment:
And if there is no `depth` tag? Why did you change the original if statement?

The original is safer:
```
if (xmldoc && xmldoc.getElementsByTagName('depth') !== undefined) {
```